### ISSUE TITLE
[test-triage] chip_csr_bit_bash

### ISSUE THREAD

**Issue by msfschaffner** [lowRISC/opentitan#15898]
### Hierarchy of regression failure

Chip Level

### Failure Description

Job chip_earlgrey_asic-sim-vcs_run_cover_reg_top killed due to: Exit reason: User job exceeded runlimit: User job timed out has 1 failures:

Test chip_csr_bit_bash has 1 failures.
2.chip_csr_bit_bash.680983478
Log /container/opentitan-public/scratch/os_regression/chip_earlgrey_asic-sim-vcs/2.chip_csr_bit_bash/latest/run.log

  Job ID: smart:dca360fb-4df9-4345-8c1d-40060dbab0a5

### Steps to Reproduce

- GitHub Revision: [53874025b](https://github.com/lowrisc/opentitan/tree/53874025b840788d5e66ffddae8fe7840a9cf4df)
- dvsim invocation command to reproduce the failure, inclusive of build and run seeds:
  ./util/dvsim/dvsim.py hw/top_earlgrey/dv/chip_sim_cfg.hjson -i chip_csr_bit_bash --fixed-seed 680983478 --build-seed 137638316 --waves -v h

**Comment by sriyerg** [comment]
Fixed in #15968.

**Comment by msfschaffner** [comment]
Passing 3 times in a row, so we can close this!

```
|    | Tests             |   2022-11-04 07:12:05 |   2022-11-03 07:11:23 |   2022-11-02 07:03:26 |   2022-11-01 07:10:18 |   2022-10-31 07:08:35 |   2022-10-30 07:09:58 |   2022-10-29 07:10:41 |   2022-10-28 07:09:55 |   2022-10-27 07:09:48 | Suite        |
|---:|:------------------|----------------------:|----------------------:|----------------------:|----------------------:|----------------------:|----------------------:|----------------------:|----------------------:|----------------------:|:-------------|
|  2 | chip_csr_bit_bash |                   100 |                   100 |                   100 |                    80 |                   100 |                   100 |                   100 |                   100 |                   100 | csr_bit_bash |
```
